frontend/app/src/util/useThemeManager.ts

```diff
@@ -100,8 +100,8 @@ export function useThemeManager(): [ThemeManager, object[]] {
     // should check and update the theme after printing if necessary.
     window.addEventListener("afterprint", updateAutoTheme)
     return () => {
-      mediaMatch.removeEventListener("change", updateAutoTheme)
       window.removeEventListener("afterprint", updateAutoTheme)
+      mediaMatch.removeEventListener("change", updateAutoTheme)
```

Comment:
@kmcgrady I have swapped these lines to fix a hash-collision issue in the build merge gate and this fixes it 😅